internal/core/bootstrap.go

```diff
@@ -45,6 +47,11 @@ var processID string
 const sessionIDPlaceholder = "testcontainers-go:%d:%d"
 
 func init() {
+	cfg := config.Read()
```

Comment:
@mdelapenya could calling `config.Read()` in `init` cause undesired behaviour due to how early it called?

I'm thinking this would prevent things like changing the environment variables in a test from having the desired effect as an example.

Reply:
Thanks for the feedback, @stevenh. I understand your concern: reading the config in an `init()` function felt wrong.

I change the code so that `config.Read` falls back to the session in `core/bootstrap` (I moved `bootstrap.go` to a new package to avoid import cycle between `internal/config` and ``internal/core`). Please let me know what you think.